setup.py

```diff
@@ -105,9 +86,10 @@ def configuration(parent_package='', top_path=None):
         ],
 
         configuration=configuration,
-        setup_requires=SETUP_REQUIRES,
         install_requires=INSTALL_REQUIRES,
-        requires=REQUIRES,
```

Comment:
But please also add a comment since apparently this was a point of confusion.  :smile: